public-api/src/main/java/org/pmiops/workbench/cdr/dao/DbDomainDao.java

```diff
@@ -1,10 +1,16 @@
 package org.pmiops.workbench.cdr.dao;
 import org.pmiops.workbench.cdr.model.DbDomain;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
 
 public interface DbDomainDao extends CrudRepository<DbDomain, Long> {
     // TODO -- maybe add order by
     List<DbDomain> findAll();
+
+    List<DbDomain> findByDbType(String db_type);
+
+    @Query(nativeQuery = true,value = "select * from db_domain where db_type=?1 and concept_id <> 0")
```

Comment:
I think just using findByDBType() will work here.   We really don't care about the concept id now and some domains we filter on may not have one.

Reply:
If you want to say concept id not 0 , you can do findByDbtypeAndConceptIdIsNotNull or something. If you want to try it out. Could make that nullable instead of zeros.